### DISCUSSION TITLE
Add pref "Automatically add these trackers to new downloads"

### DISCUSSION THREAD

**Discussion by dantearmok** [transmission/transmission#5429] [Ideas]
Sometimes torrents don't have reachable trackers.  The fix is to add a few manually - a horrible chore when you're working on a dozen new torrents.

How 'bout adding a preference (checkmark) "Automatically add trackers to new torrents:" with a field large enough to hold two or three tracker urls ?

That's what qB has; I find it to be very useful.

What do you'all think?

**Comment by GaryElshaw** [comment]
The GTK and QT clients have this function, i'd love to see it on macOS too.

**Comment by positiveque** [comment]
You can use web GUI to add trackers for all new torrents by default. Also I suggest check fresh trackers lists https://github.com/ngosang/trackerslist . Very useful.

![image](https://github.com/transmission/transmission/assets/5774711/1e487cf4-2384-4b04-92e8-957bafcd4612)
